### ISSUE TITLE
Как поменять время операции

### ISSUE THREAD

**Issue by AZEN-SGG** [AbrLis/Distributed-computing#1]
Привет, проверяю твою работу, такой вопрос, а ты реализовал возможность менять значение времени исполнения задачи

**Comment by AbrLis** [comment]
Нет, это не реализовано. Поменять можно только изменив константы в коде.

Если что можно написать в телеграм, @AbrLis

чт, 22 февр. 2024 г., 01:45 AZEN-SGG ***@***.***>:
